src/manifest.json

```diff
@@ -32,7 +32,10 @@
     "128": "icons/icon128.png"
   },
   "web_accessible_resources": ["icons/*.png"],
-  "options_page": "src/options/options.html",
-  "content_security_policy": "script-src 'self' https://cdn.jsdelivr.net 'unsafe-eval'; object-src 'self'",
+  "options_ui": {
+    "page": "options/options.html",
+    "open_in_tab": true
+  },
+  "content_security_policy": "script-src 'self' 'unsafe-eval'; object-src 'self'",
```

Comment:
We do not need `https://cdn.jsdelivr.net` in the CSP anymore thanks to https://github.com/orbit-love/orbit-browser-extension/pull/30